src/java.base/share/classes/java/util/Currency.java

```diff
@@ -1174,15 +1176,12 @@ private static boolean containsInconsistentInstances(
                     || prop.fraction != fractionDigit);
         }
 
-        private static boolean isPastCutoverDate(String s)
-                throws ParseException {
-            SimpleDateFormat format = new SimpleDateFormat(
-                    "yyyy-MM-dd'T'HH:mm:ss", Locale.ROOT);
-            format.setTimeZone(TimeZone.getTimeZone("UTC"));
-            format.setLenient(false);
-            long time = format.parse(s.trim()).getTime();
-            return System.currentTimeMillis() > time;
-
+        // cutOver adheres to ISO8601 Local Date Time format (excluding nano secs)
+        private static boolean isPastCutoverDate(String cutOver) {
+            return System.currentTimeMillis() >
+                    LocalDateTime.parse(cutOver.trim(), DateTimeFormatter.ISO_LOCAL_DATE_TIME)
```

Comment:
Btw, do we really need this `trim()` call?
It looks redundant. prop.date is result part of `m.group(4)` of this regexp:
https://github.com/openjdk/jdk/blob/3f1d9b573546685215af06031656efe6f1429caf/src/java.base/share/classes/java/util/Currency.java#L255-L257
`(\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2})`

I don't see how it could contain whitespace characters on start or end.

Reply:
That seems to be correct to me

Reply:
Right, that previous `trim()` is unnecessary. Removed.